delfin/tests/unit/drivers/dell_emc/vnx/vnx_block/test_vnx_block.py

```diff
@@ -331,3 +318,63 @@ def test_err_cli_res_to_list(self):
             navi_handler = NaviHandler(**ACCESS_INFO)
             navi_handler.cli_res_to_list({})
         self.assertIn('cli resource to list error', str(exc.exception))
+
+    @mock.patch.object(time, 'mktime')
+    def test_time_str_to_timestamp(self, mock_mktime):
+        tools = Tools()
+        log_time = '03/26/2021 14:25:36'
+        tools.time_str_to_timestamp(log_time, consts.TIME_PATTERN)
+        self.assertEqual(mock_mktime.call_count, 1)
+
+    @mock.patch.object(time, 'strftime')
+    def test_timestamp_to_time_str(self, mock_strftime):
+        tools = Tools()
```

Comment:
Same as below, please make sure that the unit test is useful.

Reply:
modifed